dogs.html

```diff
@@ -0,0 +1,231 @@
+
+            .cls-3 {
+              font-size: 62.641px;
+              font-family: WorkSans-Black, Work Sans;
+              letter-spacing: -0.009em;
+            }
+
+            .cls-3,
+            .cls-7 {
+              fill: #fff;
+            }
+
+            .cls-4 {
+              letter-spacing: -0.03em;
+            }
+
+            .cls-5 {
+              letter-spacing: -0.098em;
+            }
+
+            .cls-6 {
+              letter-spacing: 0em;
+            }
+          </style>
+          <clipPath id="clip-path" transform="translate(-1 -1.168)">
+            <rect class="cls-1" x="1" y="1.168" width="502.59" height="135.832" />
+          </clipPath>
+        </defs>
+        <title>adoptadog-logo</title>
+        <g class="cls-2"><text class="cls-3" transform="matrix(0.966, 0, -0.259, 0.966, 117.172, 133.124)">ADO<tspan class="cls-4" x="131.984" y="0">P</tspan>
+            <tspan class="cls-5" x="174.329" y="0">T</tspan>
+            <tspan x="211.162" y="0">ADO</tspan>
+            <tspan class="cls-6" x="343.146" y="0">G</tspan>
+          </text>
+          <path class="cls-7"
+            d="M126.091,35.331c-1.677-.71-8.14-3.5-9.607-4.406s-3.959-2.266-3.847-3.395a5.082,5.082,0,0,0-2.6-4.976c-2.6-1.695-7.8-4.525-8.147-5.769s.8-3.731,1.021-5.315,1.923-4.636,1.923-6.332.112-4.864-2.146-3.731-2.6,4.07-4.3,6.444-5.429,7.354-6.1,8.934-5.542,5.881-7.012,7.349-7.577,10.633-8.368,13.8-7.914,16.06-11.872,19.905-14.254,15.719-15.95,18.1-11.2,16.4-12.1,19.112a128.846,128.846,0,0,0-3.505,13.23c-.228,2.038-1.244,11.875-1.244,13.117s-5.882,4.977-9.161,3.619-13.568-5.77-15.04-7.8-8.03-4.747-6.9-3.279,5.2,7.579,7.461,9.388,12.9,6,16.854,4.977,8.6-2.6,11.2-1.586,2.486,4.754,8.028,5.433,17.756,3.277,20.359,2.6,2.823-3.621,1.921-3.956,7.123-3.282,8.595-3.282,3.049-2.037,3.165-2.828-1.246-2.035-2.6-2.035a12.273,12.273,0,0,1-4.3-.679c-.9-.451-1.355-2.374-.223-3.619s4.293-5.2,4.637-7.349-1.132-2.828.447-4.186,3.395-3.73,3.847-2.484,3.507,10.173,4.523,12.664,2.379,10.858,2.379,12.1-.456,5.651.791,5.77,7.572.56,8.028,0,2.488-1.021,3.056-1.021,4.405,1.244,4.858-.114.9-3.956-.23-4.293-2.147-2.828-2.147-3.168-2.146-13.8-2.258-15.61-.567-10.172-.567-11.307.455-10.968.567-12.775.467-7.237,1.251-9.161,1.007-5.2,1.349-7.579-1.474-4.407-2.035-6.9-2.376-5.769-1.923-7.916,2.037-9.275,3.393-10.063,4.523-1.135,7.238-.8,6.672,1.128,7.916.8a20.385,20.385,0,0,0,4.861-2.036c1.132-.8,1.586-1.924,2.488-2.6s4.33-4.007,2.035-4.979"
+            transform="translate(-1 -1.168)" />
+        </g>
+      </svg>
+      <h1>Seatles's premiere dog adoption agency</h1>
+    </div>
+    <div>
+      <a class="checkout-page-link" href="checkout.html">Checkout</a>
+    </div>
+  </header>
+
+  <nav>
+    <ul>
+      <li><a href=index.html>Home</a></li>
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="blog.html">Blog</a></li>
+    </ul>
+  </nav>
+  <section>
+    <h2>Here are all of our Dogs!</h2>
+  </section>
+  <section class="boxes">
```

Comment:
boxes isn't maybe the best descriptor, try to use something semantic, like "dog-box-wrapper" or something